docker/env/.lido_terra.testnet.env

```diff
@@ -11,4 +11,5 @@ ADDRESSES_REWARDS_DISPATCHER_CONTRACT=terra16c4qpu6xg2l399dr7nf2xcuscqkg22pcnn5z
 GRAFANA_PORT=3000
 SERVICE_NAME=lido_terra_testnet
 DELEGATIONS_DISTRIBUTION_CONFIG_NUM_MEDIAN_ABSOLUTE_DEVIATIONS=3
-NETWORK_GENERATION=columbus-5
\ No newline at end of file
+NETWORK_GENERATION=columbus-5
+MISSING_VOTES_GOVERNANCE_MONITOR_MONITORED_VALIDATORS=terra1v5hrqlv8dqgzvy0pwzqzg0gxy899rm4kdn0jp4,terra123gn6j23lmexu0qx5qhmgxgunmjcqsx8g5ueq2,terra15zcjduavxc5mkp8qcqs9eyhwlqwdlrzy6anwpg,terra1v5hrqlv8dqgzvy0pwzqzg0gxy899rm4kdn0jp4,terra1kprce6kc08a6l03gzzh99hfpazfjeczfpd6td0,terra1c9ye54e3pzwm3e0zpdlel6pnavrj9qqvqgf7ps,terra144l7c3uph5a7h62xd8u5et3rqvj3dqtvve3he0,terra1542ek7muegmm806akl0lam5vlqlph7spfs99vq,terra1sym8gyehrdsm03vdc44rg9sflg8zeuqwfd3384,terra1khfcg09plqw84jxy5e7fj6ag4s2r9wqsg5jt4x,terra15urq2dtp9qce4fyc85m6upwm9xul30496lytpd,terra1alpf6snw2d76kkwjv3dp4l7pcl6cn9uytq89zk,terra1nwrksgv2vuadma8ygs8rhwffu2ygk4j24pxxx0,terra175hhkyxmkp8hf2zrzka7cnn7lk6mudtv4nsp2x,terra13g7z3qq6f00qww3u4mpcs3xw5jhqwraswv3q3t,terra1jkqr2vfg4krfd4zwmsf7elfj07cjuzss3qsmhm,terra15cupwhpnxhgylxa8n4ufyvux05xu864jcrrkqa
```

Comment:
isn't it better to query the validators registry contract directly? https://finder.terra.money/columbus-5/address/terra16j67029v236npntxjatwena6kh7rgvkwdkw0tm

```
{"get_validators_for_delegation": {}}
```

Reply:
And, I guess, we could make an environment variable to configure address of contract to query